source/js/buyers-guide/components/filter/filter.jsx

```diff
@@ -216,10 +230,12 @@ export default class Filter extends React.Component {
   }
 
   setVisibilities() {
+    const CLASS_HIDDEN = `d-none`;
```

Comment:
as constants, let's lift these all the way up (probably even to outside the component class), rather than declaring them every time setVisibilities is called.

Reply:
good call!